utilities.php

```diff
@@ -872,8 +872,8 @@ public static function get_share_counts_api_status($response) {
     // Did it return at least 8 services?
     $has_majority_services = count(array_keys($response['body']['data'])) >= 8 ? true : false;
```

Comment:
Makes sense! We know some of these services are not 100% reliable. Let's make the check a bit looser ie. reduce the threshold to 6 or 7 (for twitter + delicious)